app/models/reports.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+require_relative "reports/monthly_bulk_serializer"
```

Comment:
Already had a chat about this. Happy to accept we don't know why this is needed in models, but not in presenters or serializers. But could we add a comment owning up to this, something like

```ruby
# We have put everything to do with reports into a `/reports` folder in the
# model, presenters, and services folder mainly just to keep things tidy.
# For presenters and services that was fine, but things broke in models.
#
# Hence this file, which magically seems to fix things! We don't understand why,
# and should we ever get 5 minutes and the inclination to do so we might try
# figure it out. But for now this we're just moving on 😁
```

Reply:
```suggestion
# We have put everything to do with reports into a `/reports` folder in the
# model, presenters, and services folder mainly just to keep things tidy.
# For presenters and services that was fine, but things broke in models.
#
# Hence this file, which magically seems to fix things! We don't understand why,
# and should we ever get 5 minutes and the inclination to do so we might try
# figure it out. But for now, we're just moving on 😁
require_relative "reports/monthly_bulk_serializer"
```